core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -340,6 +340,10 @@
   MASTER_REPLICATION_COORDINATOR_THREADCHECK("master.replication.coordinator.threadcheck.time",
       "5s", PropertyType.TIMEDURATION,
       "The time between adjustments of the coordinator thread pool"),
+  /**
+   * @deprecated since 1.9.1
```

Comment:
I actually meant in the text in the property description. The javadoc comment here won't really be useful. Also, it should be 1.9.2.

Reply:
oh yeah, should be be 1.9.2.  Not exactly sure what you are looking for, can you give a code example?

Reply:
I updated the docs.   I looked into the the code that generates prop docs and it generates documentation for deprecated props.  So users looking at the docs will know which props are deprecated, but they will not know when.  If [ACCUMULO-4592](https://issues.apache.org/jira/browse/ACCUMULO-4592) were done, could also add a deprecated version.